_docs/setup/kubernetes/aws-nlb.md

```diff
@@ -0,0 +1,92 @@
+---
+title: AWS NLB
+overview: Instructions to use and configure ingress istio with aws NLB.
+
+order: 12
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Instructions to use and configure ingress istio with aws NLB.
+
+## Prerequisites
+The following instructions require you have access to a Kubernetes **1.9.0 or newer** cluster
+*Warning:* This is an alpha feature and not recommended for production clusters yet.
+
+
+## IAM Policy
+You need to apply policy in the master roles in order to be able to provition network load balancer
+
+```JSON
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "kopsK8sNLBMasterPermsRestrictive",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVpcs",
+                "elasticloadbalancing:AddTags",
+                "elasticloadbalancing:CreateListener",
+                "elasticloadbalancing:CreateTargetGroup",
+                "elasticloadbalancing:DeleteListener",
+                "elasticloadbalancing:DeleteTargetGroup",
+                "elasticloadbalancing:DescribeListeners",
+                "elasticloadbalancing:DescribeLoadBalancerPolicies",
+                "elasticloadbalancing:DescribeTargetGroups",
+                "elasticloadbalancing:DescribeTargetHealth",
+                "elasticloadbalancing:ModifyListener",
+                "elasticloadbalancing:ModifyTargetGroup",
+                "elasticloadbalancing:RegisterTargets",
+                "elasticloadbalancing:SetLoadBalancerPoliciesOfListener"
+            ],
+            "Resource": [
+                "*"
+            ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVpcs",
+                "ec2:DescribeRegions"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+## Rewrite Istio Ingress Service
```

Comment:
I think this section will make this document hard to maintain and keep people from trying this work out.

If you do take the approach in this document, I'd recommend a blog post.

Another approach is to integrate into the main helm chart here:
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/ingress/templates/service.yaml

Keep in mind the network WG is rapidly working on a new ingress/egress model (called ingressgateway and egressgateway).  I am not up to speed on the plans of that work.